modules/cyclone-dx/src/main/java/com/eclipse/sw360/antenna/cyclonedx/CycloneDXGenerator.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Robert Bosch Manufacturing Solutions GmbH 2020.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ * <p>
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.eclipse.sw360.antenna.cyclonedx;
+
+import org.cyclonedx.BomGenerator;
+import org.cyclonedx.BomGeneratorFactory;
+import org.cyclonedx.CycloneDxSchema;
+import org.cyclonedx.model.Bom;
+import org.cyclonedx.model.Component;
+import org.eclipse.sw360.antenna.api.Attachable;
+import org.eclipse.sw360.antenna.api.IAttachable;
+import org.eclipse.sw360.antenna.api.configuration.ToolConfiguration;
+import org.eclipse.sw360.antenna.api.exceptions.ExecutionException;
+import org.eclipse.sw360.antenna.api.workflow.AbstractGenerator;
+import org.eclipse.sw360.antenna.model.artifact.Artifact;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Document;
+
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.*;
+import java.nio.file.Path;
+import java.util.*;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Creates a CycloneDX Bill-of-Material file with information from Antenna.
+ */
+public class CycloneDXGenerator extends AbstractGenerator {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CycloneDXGenerator.class);
+
+    @Override
+    public Map<String, IAttachable> produce(Collection<Artifact> artifacts) throws ExecutionException {
+        LOG.info("Created a cyclone-dx bill-of-material for {} artifacts", artifacts.size());
```

Comment:
Please only debug logging in the component.

Reply:
You are aware that information will then be lost in a normal run from Maven? Anyway, if all components should log on debug, then this should not stand out. Changing it to debug accordingly.

Reply:
Yeah, but we actually want the normal run to be quite clean and only in case of debug run to get more information.